packages/dds/tree/src/feature-libraries/README.md

```diff
@@ -1,12 +1,140 @@
+    Some options are listed here, but little thought has gone into this list:
+
+    -   typescript DSL (like [TypeBox](https://www.npmjs.com/package/@sinclair/typebox)) that supports runtime and compile type typing without code gen.
+    -   [json schema](https://json-schema.org/)
+    -   graphQL
+
+-   Schematize configurations:
+    Likely the only schematize configuration we will want is one for using it with `Schema` and `EditableTree`, but other options are possible.
+    Likely will want optional fast path for when the `EditableTree` is backed by a `Forest` using shape based compression: may be able to skip some or all validation for whole subtrees this way.
+
+-   Schematize Types: A typescript type-meta-function that subsets the `EditableTree` API to be schema aware (for both reading and editing)
+
+-   Normalizers: Libraries to convert EditableTrees into representations ideal for use in specific frameworks/platforms/tool-kits. Like schematize, may have a fast-path for specific compressed data representations.
+
+    -   react
+    -   custom?
+
+## Possible System / Service Configurations
+
+Several different DDS setups can be made with these libraries.
+
+For example, alfred (or some other service) could run a `Rebaser` sufficient to detect all conflicts, marking (or discarding) conflicted ops.
+Such a service could also optionally enforce some validation or editing permissions.
+Or conflict resolution can run fully client side.
+A service could run a `Rebaser` that outputs a stream of tree Diffs and saved tree snapshot summaries allowing lighter weight clients, and optionally granular read permissions.
+Services could provide additional indexes (such as history or search).
+
+Clients may or may not (depending on their `Rebaser` setup) be able to run with no actual copy of the tree in the DDS at all.
+
+TODO: Enumerate some possible configurations and their motivations. Unify this with the list of use-cases below somehow.
+
+## Use Cases
+
+TODO: this section is just a collection of unorganized notes, and needs some work to be properly integrated into this document.
+
+### Systems with existing data
+
+There are a few different usage patterns for systems with existing data:
+
+-   Import / Copy / Convert data into Fluid system. Maybe leave old one readonly.
+-   Re-implement existing APIs / services / apps using Fluid for storage (can add Fluid as option and/or migrate data some at a time (ex: document level like whiteboard is doing), or migrate whole service).
+-   Leave existing system as source of truth, and add optional Fluid front end with bidirectional sync.
+-   Use Fluid as new source of truth, but support bidirectional sync to the other system.
+-   Have an application that works with both existing data and Fluid data (may want to have compatible type/schema systems)
+
+### Misc use-case
```

Comment:
I think many of the items in this section are a bit cryptic. If we want this section to be generally understandable by contributors rather than just personal notes/reminders for the authors of the section, I we should describe most of these use cases in at least slightly more detail.

Reply:
I agree, but thats not in scope for this particular work (which is just moving this list between docs). The TODO above is intended to reflect that this work is still needed.